config/testdata/example_config.yaml

```
@@ -19,6 +19,10 @@ athenz:
   url: https://www.athenz.com/zts/v1
   timeout: 5s
   root_ca: _root_ca_
+x509:
+  cert: ""
+  key: ""
+  poll_interval: ""
```

```suggestion
athenz:
  tls/mtls/client_cert:
    cert: ""
    key: ""
    poll_interval: ""
```
I think it is better to put under Athenz struct.

---

```yaml
athenz:
  auth_header: Athenz-Principal-Auth
  url: https://www.athenz.com/zts/v1
  timeout: 5s
  cert: "" # new
  key: "" # new
  cert_key_poll_interval: "" # new
  root_ca: _root_ca_
```

---

my bad. I will just do `poll_interval` instead. `cert_key_poll_interval` sounds weird.

---

Handled in https://github.com/AthenZ/garm/pull/50/commits/67e2e38fd1e4f9ff365cbca307d3657ae63f2330